executor/executor.go

```diff
@@ -128,6 +128,15 @@ func (e *baseExecutor) Schema() *expression.Schema {
 	return e.schema
 }
 
+// inlineProjection supports to prune unneeded columns for outputs of a executor.
+func (e *baseExecutor) inlineProjection() (childrenUsedSchema [][]bool) {
```

Comment:
I think `inlineProjection` is not a good name for the functionality of this function. 
Can we rename it to `markUnprunedChildCol` and rename `childrenUsedSchema ` to `unprunedFlags`?

Reply:
I think `used` is better for human understading. It's an affirmative voice, and `unpruned` is a negetive voice which makes people think.
And all the voice is affirmative in `rule_column_pruning.go`.

Reply:
And I think `inlineProjection` is better here, to describe this feature as a proper noun

Reply:
But actually this func does not prune anything, `inlineProjection` makes the readers confused.

Reply:
Updated.